ops/charm.py

```diff
@@ -489,6 +685,13 @@ def from_yaml(
 
 
 class RelationRole(enum.Enum):
+    """An annotation for a Charm's Role in a Relations
+
+    For each relation a Charms role may be
+    - A Peer
+    - A consumer in the relation ('requires')
```

Comment:
Except not always I suppose haha

Reply:
Changed to "service provider" and "service consumer"